File: P1A9_Empleados/Python/Empleado/Empleado.py
class Empleado:
    # Constructores (Simulado en Python con argumentos por defecto)
    def __init__(self, nombre="", anio_contratacion=0, salario=0.0):
        # Atributos privados (convención en Python usando _)
        self._nombre = nombre
        self._anio_contratacion = anio_contratacion
        self._salario = salario

    # Getters y Setters
    def get_nombre(self):
        return self._nombre

    def set_nombre(self, nombre):
        self._nombre = nombre

    def get_anio_contratacion(self):
        return self._anio_contratacion

    def set_anio_contratacion(self, anio_contratacion):
        self._anio_contratacion = anio_contratacion

    def get_salario(self):
        return self._salario

    def set_salario(self, salario):
        self._salario = salario

    # Metodos
    def calcular_salario(self):
        return self._salario

    def __str__(self):
        return f"\nNombre: {self._nombre}\nAnio de Contratacion: {self._anio_contratacion}\nSalario: {self._salario}"

File: P1A9_Empleados/C++/Archivos_Origen/empleado.cpp
#include "../Archivos_Encabezado/empleado.h"
using namespace std;

//Definir los constructores
empleado::empleado(){
    this->nombre = "";
    this->anioDeContratacion = 0;
    this->salario = 0.0;
}

empleado::empleado(string n, int a, double e){
    this->nombre = n;
    this->anioDeContratacion = a;
    this->salario = e;
}

//Getters y setters
string empleado::getNombre(){
    return this->nombre;
}

int empleado::getAnioDeContratacion(){
    return this->anioDeContratacion;
}

double empleado::getSalario(){
    return this->salario;
}

void empleado::setNombre(string n){
    this->nombre = n;
}

void empleado::setAnioDeContratacion(int a){
    this->anioDeContratacion = a;
}

void empleado::setSalario(double e){
    this->salario = e;
}

//Metodos

double empleado::calcularSalario(){
    return this->salario;
}

string empleado::toString(){
    return "Nombre: " + this->nombre + "\nAnio de contratacion: " + to_string(this->anioDeContratacion) + "\nSalario:" + to_string(this->salario);
}
